.ci-dockerfiles/ubuntu-openssl-1.1.0/README.md

```diff
@@ -0,0 +1,28 @@
+# Build image
+
+```bash
+docker build -t ponylang/ponyc-ci:ubuntu-openssl_1.1.0 .
+```
+
+# Run image to test
+
+Will get you a bash shell in the image to try cloning Pony into where you can test a build to make sure everything will work before pushing:
+
+```bash
+docker run --name ponyc-ci-ubuntu-openssl_1.1.0 --user pony --rm -i -t ponylang/ponyc-ci:unbuntu-openssl_1.1.0 bash
+```
+
+# Run CircleCI jobs locally
+
+Use the [CircleCI CLI](https://circleci.com/docs/2.0/local-cli/) to run the CI job using this image
+from the ponyc project root:
+
+```bash
+circleci build --job ubuntu-openssl-110
```

Comment:
this is missing the closing quotes and part of the instructions for running locally

Reply:
Fixing